components/raftstore/src/store/fsm/peer.rs

```diff
@@ -6555,6 +6557,58 @@ where
     fn register_report_region_buckets_tick(&mut self) {
         self.schedule_tick(PeerTick::ReportBuckets)
     }
+
+    fn register_check_peer_complete_recovery(&mut self) {
+        self.schedule_tick(PeerTick::CheckPeerCompleteApplyLogs)
+    }
+
+    fn on_check_peer_complete_apply_logs(&mut self) {
```

Comment:
why not call it in `on_apply_res` which indicates apply index is probably updated?

Reply:
Accepted. Thx.